javascript/javascript3/week1/gifAPI.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Gif Api</title>
+</head>
+<style>
+    #btn{
+    height:50px;
+    border-radius: 7px;
+    width:180px;
+    background-color: rgb(7, 39, 107);
+    color: white;
+}
+#container-input{
+    text-align: center;    
+}
+
+#input-textbox{
+    height :40px;
+    border-radius: 5px;
+    margin-bottom: 1%;
+    margin-top:1%;
+    width: 200px;
+}
+
+#input-number{
+    height :40px;
+    border-radius: 5px;
+    margin-bottom: 1%;
+    margin-top:1%;
+    width: 200px;
+}
+
+</style>
+
+<body>
+ <h1>Giffy Searcher</h1>
+
+ <div id="container-input">
+    <input type="text" id="input-textbox" placeholder="Search for gifs" required><br>
+    <button type="button" id="btn">Search for gifs here!</button><br>
+    <input type="text" id="input-number" placeholder="Number of gifs" required>
+    <ul id =gifImage></ul>
```

Comment:
you forgot quotes in `gifImage`, and why is the name in camelCase? Use kebab-case, as for other classes and ids